### PR TITLE
[Workers] Adds New Python Examps

### PR DESCRIPTION
**Pull request by megaconfidence**
This PR adds new Python examples for:
- [Hot-link protection](https://developers.cloudflare.com/workers/examples/hot-link-protection/)
- [Logging headers to console](https://developers.cloudflare.com/workers/examples/logging-headers/)
- [Modify request property](https://developers.cloudflare.com/workers/examples/modify-request-property/)
- [Post JSON](https://developers.cloudflare.com/workers/examples/post-json/)
- [Read POST](https://developers.cloudflare.com/workers/examples/read-post/)

### CODE REVIEW

File: content/workers/examples/logging-headers.md [hunk 1]
```diff
@@ -42,6 +43,17 @@ export default {
 } satisfies ExportedHandler;
 ```
 
+{{</tab>}}
+{{<tab label="py">}}
+
+```py
```
Comment (kodster28) [inline content/workers/examples/logging-headers.md:49]: Make sense to pull in additional info from [another example](https://developers.cloudflare.com/workers/languages/python/examples/#emit-logs-from-your-python-worker) in our docs?

And then remove that one from the separate "Python" examples page?

**Review by kodster28 — APPROVED** [review]
One suggestion, but looks good otherwise from a content PoV

File: content/workers/examples/read-post.md [hunk 2]
```diff
@@ -135,5 +136,40 @@ export default {
...
+    if "form" in content_type:
+        form = await request.formData()
+        data = Object.fromEntries(form.entries())
+        return JSON.stringify(data)
+    return await request.text()
+
+async def on_fetch(request):
+    def raw_html_response(html):
+        headers = Headers.new({"content-type": "text/html;charset=UTF-8"}.items())
+        return Response.new(html, headers=Headers)
```
Comment (KianNH) [inline content/workers/examples/read-post.md]:
```suggestion
        headers = Headers.new({"content-type": "text/html;charset=UTF-8"}.items())
        return Response.new(html, headers=headers)
```

File: content/workers/examples/logging-headers.md (hunk 1 above)
Comment (megaconfidence) [reply content/workers/examples/logging-headers.md:49]: They seem to be very different examples IMO, although they are both logging related

**Pull request merged**